config/environments/common/production.rb

```diff
@@ -104,23 +104,17 @@
   # config.active_record.database_resolver = ActiveRecord::Middleware::DatabaseSelector::Resolver
   # config.active_record.database_resolver_context = ActiveRecord::Middleware::DatabaseSelector::Resolver::Session
 
-  # CUSTOM
-  config.action_mailer.smtp_settings = {
-
-    user_name: Setting.sendgrid_username,
-    password: Setting.sendgrid_password,
-    domain: Setting.sendgrid_domain,
-    address: 'smtp.sendgrid.net',
-    port: 587,
-    authentication: :plain
-  }
+  # Send and receive email with Postmark
+  config.action_mailer.delivery_method = :postmark
+  config.action_mailbox.ingress = :postmark
 
-  config.mailer = {
-    from: Setting.sendgrid_from
+  config.action_mailer.postmark_settings = {
+    api_token: Setting.postmark_api_token
   }
 
-  config.action_mailbox.ingress = :sendgrid
-  config.project_name = ENV['HUNDRED_EYES_PROJECT_NAME']
+  config.action_mailer.default_options = {
+    from: "#{Setting.project_name} <#{Setting.email_from_address}>"
```

Comment:
This is not directly related to the Postmark change. But I thought it might be a good idea to reduce redundancy and simply use the project name as the from name. Also, the from address isn’t really provider-dependant, right?